schema229/markdown.py

```diff
@@ -126,4 +146,4 @@ def write_dir(input_dir_path, output_dir_path):
   if len(sys.argv) == 2:
     write_file(os.path.join(source_dir_path,f'{sys.argv[1]}.schema.yaml'), os.path.join(docs_dir_path,f'{sys.argv[1]}.schema.md'))
   elif len(sys.argv) == 1:
-    write_dir(source_dir_path, docs_dir_path)
\ No newline at end of file
+    write_dir(source_dir_path, docs_dir_path)
```

Comment:
Charlie, this looks great but we also need the fourth field "RMR Test". For the greatest flexibility it should have the following enumerations: False (attribute not used in the RMR test), Secondary (attribute is used as part of the RMR test case description but is not the target of the RMR test) and Primary (attribute is the target of RMR test). Just like the other three new fields, it will have no entiry by default which would be equivalent to False.

It is my understanding that once  this change is committeed, I will be able to create a branch off the master will these fields added and could start filling them in. Correct? Thank you!

Reply:
@mkarpman  So just to make sure:
You want the four additional columns below in this order?

RMR Test: False, Primary, Secondary
AppG Used By TCDs: True, False 
AppG Used By TCDs: True, False 
AppG B_RMR Equals P_RMR: True, False

Reply:
@mkarpman You can actually create a branch and start filling in these fields any time, but you won't see your fields in the markdown tables until this pull request is merged in.

Reply:
Is the RMR Test possibly ruleset specific? In other words should it be "AppG RMR Test"?

Reply:
I think majority/all will be generic as were were trying to avoid using any ruleset-specific elements (e.g., space conditioning category) in the App G TCDs and RDSs, but time will show. If this comes up, we could add another column to differentiate the generic tests from App G - specific tests, so I think having a single RMR Test column is good for now.

Reply:
@mkarpman I have added RMR Test to the Data Group tables.  As before, you can view the markdown at https://github.com/open229/ruleset-model-report-schema/blob/adding_data_element_fields/docs229/ASHRAE229.schema.md

Reply:
Thanks!